src/tvheadend/AutoRecordings.cpp

```diff
@@ -254,6 +254,11 @@ PVR_ERROR AutoRecordings::SendAutorecAddOrUpdate(const kodi::addon::PVRTimer& ti
       htsmsg_add_s32(m, "start", startWindowBegin);
       htsmsg_add_s32(m, "startWindow", startWindowEnd);
     }
+    else
+    {
+      htsmsg_add_s32(m, "start", -1);
+      htsmsg_add_s32(m, "startWindow", -1);
+    }
```

Comment:
As the note in line 238 says, not sending these values (like we did before), will set start and starwindow to "any" time.

Respective tvh code:

```c
      if(htsmsg_get_s32(in, "start", &start))
        start = -1;
      if(htsmsg_get_s32(in, "startWindow", &start_window))
        start_window = -1;
```

So, from my point of view, your code change changes actually nothing on server side. Not sure how this could fix something, sorry.

Reply:
And related code and logic for both the add-on and server is identical for Matrix/Nexus and tvh 4.2/4.3, I've just checked that.

Reply:
What you're stating is true for new(!) timers, not when updating existing timers (see my comment below) which don't have "any start time"-enabled.

Reply:
correct, just spotted this as well